apps/frontend/src/utils/helpers.ts

```diff
@@ -37,6 +38,11 @@ export const isMainnet = () => process.env.REACT_APP_NETWORK === 'mainnet';
 export const getServicesConfig = () =>
   servicesConfig[isMainnet() ? 'mainnet' : 'testnet'];
 
+export function dateFormat(timestamp: number) {
```

Comment:
```suggestion
export const dateFormat = (timestamp: number) => {
```
to make everything consistent